src/State/State.cpp

```diff
@@ -73,17 +73,17 @@ namespace mmfs
             if (sensorOK(sensors[i]))
             { // not nullptr and initialized
                 sensors[i]->update();
-                Wire.beginTransmission(0x42); // random address for testing the i2c bus
-                byte b = Wire.endTransmission();
-                if (b != 0x00)
-                {
-                    Wire.end();
-                    Wire.begin();
-                    recordLogData(ERROR, "I2C Error");
-                    sensors[i]->update();
-                    delay(10);
-                    sensors[i]->update();
-                }
+                // Wire.beginTransmission(0x42); // random address for testing the i2c bus
+                // byte b = Wire.endTransmission();
+                // if (b != 0x00)
+                // {
+                //     Wire.end();
+                //     Wire.begin();
+                //     recordLogData(ERROR_, "I2C Error");
+                //     sensors[i]->update();
+                //     delay(10);
+                //     sensors[i]->update();
+                // }
```

Comment:
Why was this check commented out?

Reply:
A few reasons: 
- It is only there because the BNO055 specifically sucks
- We were reworking the record data library
- I would have had to write a `Wire` stub and this was easier. 

I can probably uncomment it if we think we need it. probably depends on whether or not anyone plans to use the BNO055. Maybe it should be specifically added to the BNO055 update method instead of here.

Reply:
Made an issue for it